omnibus_overrides.rb

```diff
@@ -1,16 +1,13 @@
 build_iteration 1
 
-# Using pins that agree with chef 13.11.3.
-override :chef,           version: "v14.12.3"
-override :ohai,           version: "v14.8.11"
+override :chef,           version: "v15.10.12"
+override :ohai,           version: "v15.9.1"
 
 override :bundler,        version: "1.17.3"
-override :rubygems,       version: "2.7.9"
```

Comment:
pretty sure this is all ignored at this point since we removed the rubygems / bundler dep from ruby